src/meta-srv/src/failure_detector.rs

```diff
@@ -67,13 +77,28 @@ impl Default for PhiAccrualFailureDetector {
             min_std_deviation_millis: 100_f32,
             acceptable_heartbeat_pause_millis: 3000,
             first_heartbeat_estimate_millis: 1000,
-            heartbeat_history: HeartbeatHistory::new(1000),
-            last_heartbeat_millis: None,
         }
     }
 }
 
+impl Default for PhiAccrualFailureDetector {
+    fn default() -> Self {
+        PhiAccrualFailureDetector::from_options(PhiAccrualFailureDetectorOptions::default())
```

Comment:
Perhaps `Self::from_options(Default::default())`. I don't have a strong preference and don't know the style preferred in greptimedb.

Reply:
Both are fine, we don't have a style guide on this either. Though I personally prefer `Self`.

Reply:
```suggestion
        Self::from_options(Default::default())
```

Yes. I notice that `Self` is preferred in most of this codebase.